data/sql_incremental_aggregates/prepare_schedule_a_aggregate_size.sql

```diff
@@ -33,6 +33,10 @@ create index on ofec_sched_a_aggregate_size_tmp (size);
 create index on ofec_sched_a_aggregate_size_tmp (total);
 create index on ofec_sched_a_aggregate_size_tmp (count);
 
+-- this drops totals during rebuild
+drop table if exists ofec_sched_a_aggregate_state cascade;
+drop table if exists ofec_sched_a_aggregate_size_old cascade;
```

Comment:
Oh, no, what I meant was we have these two statements right below these new `drop table` statements:

```sql
alter table if exists ofec_sched_a_aggregate_size rename to ofec_sched_a_aggregate_size_old;
alter table ofec_sched_a_aggregate_size_tmp rename to ofec_sched_a_aggregate_size;
```

But in reading them again without tired eyes, I see what you mean and these are working against different tables.  Please disregard, sorry!